src/wap_proto.bnf

```diff
@@ -76,12 +76,13 @@ The following ABNF grammar defines the Wallet Access Protocol:
     ;  Wallet asks daemon to start mining. Daemon replies with START-OK, or  
     ;  ERROR.                                                                
 
-    START           = signature %d11 start height
-    start height    = number-8              ; 
+    START           = signature %d11 start_height
+    start height    = number-4              ; 
 
     ;  Daemon replies to a start mining request.                             
 
-    START-OK        = signature %d12
+    START-OK        = signature %d12 curr_height
+    curr height     = number-4              ; 
 
```

Comment:
It was a bug in the zproto BNF generator that I just fixed.

On Sun, Jan 4, 2015 at 4:27 PM, oranjuice <notifications@github.com> wrote:

> In src/wap_proto.bnf
> <https://github.com/oranjuice/monero-0mq/pull/4#discussion-diff-22440000>:
>
> >
> >      ;  Daemon replies to a start mining request.
> >
> > -    START-OK        = signature %d12
> > +    START-OK        = signature %d12 curr_height
> > +    curr height     = number-4              ;
> >
>
> was it a mistake before that the underscore was missing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oranjuice/monero-0mq/pull/4/files#r22440000>.
>